src/ducks/classifications.js

```diff
@@ -51,20 +54,16 @@ const classificationReducer = (state = initialState, action) => {
       return Object.assign({}, state,{
         status: CLASSIFICATION_STATUS.SENDING,
       });
-
+      
     //Submitting Classification also resets the store.
-    case SUBMIT_CLASSIFICATION_SUCCESS:
+    //This is only called once the WHOLE queue of Classifications has been processed.
+    case SUBMIT_CLASSIFICATION_FINISHED:
```

Comment:
I can't find where this is called.

Reply:
Hot damn, you're right, I didn't put that in.